packages/twenty-front/src/modules/workflow/workflow-diagram/components/WorkflowDiagramBaseStepNode.tsx

```diff
@@ -21,7 +31,7 @@ const StyledStepNodeType = styled.div`
     ${({ theme }) => theme.border.radius.sm} 0 0;
 
   color: ${({ theme }) => theme.font.color.light};
-  font-size: ${({ theme }) => theme.font.size.md};
+  font-size: 9px;
```

Comment:
Thomas wants these pixel-perfect values. We can replace these hard-coded values with tokens from the design system later.